src/components/Todolist.tsx

```diff
@@ -1,10 +1,86 @@
-import React from 'react';
-
-export const Todolist = () => {
-    return (
-        <div>
-
-        </div>
-    );
-};
-
+import React, { ChangeEvent, KeyboardEvent, useState } from 'react';
+import { FilterType, TasksType } from '../App';
+import './Todolist.css';
+
+export type TodolistType = {
+  title: string;
+  tasks: TasksType[];
+  removeTask: (id: string) => void;
+  filterTask: (filter: FilterType) => void;
+  addTask: (title: string) => void;
+  changeCheckStatus: (isDone: boolean, id: string) => void;
+  filter: FilterType;
+};
+export const Todolist = (props: TodolistType) => {
+  const [title, setTitle] = useState('');
+  const [eror, serEror] = useState<null | string>();
```

Comment:
Тут опечатки – и лучше использовать undefined вместо null где можно, плюс лучше ставить undefined в конец списка типов в объединениях типов

```suggestion
  const [error, serError] = useState<string | undefined>();
```